UCX compilation error with CLANG 5

/src/uct/ib/ud/base/ud_iface.c:7:
/src/uct/ib/ud/base/ud_iface.h:198:46: error: taking address of packed member 'queue' of class or structure 'uct_ud_send_skb' may result in an unaligned pointer
      value [-Werror,-Waddress-of-packed-member]
        ucs_queue_push(&iface->tx.res_skbs, &skb->queue);
                                             ^~~~~~~~~~

--- a/src/uct/ib/ud/base/ud_def.h
+++ b/src/uct/ib/ud/base/ud_def.h
@@ -7,12 +7,12 @@
 #ifndef UD_DEF_H_
 #define UD_DEF_H_
 
-
-#include <ucs/sys/math.h>
-
+#include <uct/ib/base/ib_iface.h>
+#include <ucs/arch/cpu.h>
 #include <ucs/datastruct/queue.h>
 #include <ucs/datastruct/frag_list.h>
-#include <uct/ib/base/ib_iface.h>
+#include <ucs/sys/math.h>
+
 
 #define UCT_UD_QP_HASH_SIZE     256
 #define UCT_UD_TX_MODERATION    64
@@ -146,7 +146,7 @@ typedef struct uct_ud_send_skb {
     uint8_t                 flags;
     int8_t                  status;     /* used in case of failure */
     uct_ud_neth_t           neth[0];
-} UCS_S_PACKED uct_ud_send_skb_t;
+} UCS_S_PACKED UCS_V_ALIGNED(UCS_SYS_CACHE_LINE_SIZE) uct_ud_send_skb_t;
 
 
 typedef struct uct_ud_comp_desc {

